UPGRADE-3.x.md

```diff
@@ -1,6 +1,28 @@
 UPGRADE 3.x
 ===========
 
+## Added constant for show/list field types and deprecated some field types.
```

Comment:
```suggestion
## Added constants for "show" and "list" field types
```
BTW. being strict, given the class where these constants were added this message maybe should say "templating types" instead of "field types".

Reply:
@phansys Done !